### PR TITLE
Fixes #9454 - Add axis labels and reposition legend in charts

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

**Comment by tbrisker** [comment]
[test]

File: LICENSE [hunk 1]
```diff
@@ -32,6 +32,7 @@ The following files and directories are exceptions:
 * vendor/assets/javascripts/diff.js is under Revised BSD license
 * vendor/assets/javascripts/jquery.cookie.js is GNU GPL v2/MIT licensed
 * vendor/assets/javascripts/jquery.dataTables.min.js is GNU GPL v2/BSD
+* vendor/assets/javascripts/jquery.flot.axislabels.js is MIT licensed
```
Comment (lzap) [inline LICENSE]: It is GNU GPLv3 and MIT.

**Comment by lzap** [comment]
LGTM small comment, then I will [test]

**Comment by dLobatog** [comment]
@tbrisker Screenshot with the changes? I think I still see the legend covering the Resources graph on host.

**Comment by tbrisker** [comment]
Before:
host_before
run_dist_before

After:
host_after
run_dist_after

There is a small overlap in the resources graph on the host, this is because the legend is too long.
I can make it split into two columns, but I'm not sure if this would look better:
host_opt

**Comment by dLobatog** [comment]
Merged as 6b7cb98f57cee7e116b82b59fd8100fc6f59f16d, thanks @tbrisker!

**Pull request closed without merging**